text/machine_comprehension/shufflenet_v2/README.md

```diff
@@ -0,0 +1,59 @@
+# ShuffleNetV2
+
+## Use-cases
+Computationally efficient CNN architecture designed specifically for mobile devices with very limited computing power.
+
+## Description
+[ShuffleNetV2](https://pytorch.org/hub/pytorch_vision_shufflenet_v2/) is an architecture that is the state-of-the-art in terms of speed and accuracy tradeoff.
+
+## Model
+
+|Model        |Download  |Checksum| Download (with sample test data)|ONNX version|Opset version|Top-1 error |Top-5 error |
+|-------------|:--------------|:--------------|:--------------|:--------------|:--------------|:--------------|:--------------|
```

Comment:
You can put the Checksum directly in the table like: https://github.com/onnx/models/tree/master/vision/style_transfer/fast_neural_style